backend/Dockerfile

```diff
@@ -44,10 +44,18 @@ WORKDIR ${DOCKER_WORKDIR}
 
 RUN mkdir -p /database
 
+##################################################################################
+# BASE with build tools for rust based npm modules like @iota/client #############
+##################################################################################
+FROM base as base_extended 
+
+# needed for building @iota/client, it is a rust based module
+RUN apk add --no-cache rust cargo python3 make g++
+
```

Comment:
I have created a new stage, to prevent duplicate code. 
But if you like it more I could add the line in line 53 and 69.
For the release build it would be needed too.